forge/files/src/index.js

```diff
@@ -1,32 +1,15 @@
 /* eslint-disable no-console */
-const { app, BrowserWindow, protocol } = require('electron');
-const { dirname, join, resolve } = require('path');
-const protocolServe = require('electron-protocol-serve');
-const isDev = require('electron-is-dev');
 const { default: installExtension, EMBER_INSPECTOR } = require('electron-devtools-installer');
+const { pathToFileURL } = require('url');
+const { app, BrowserWindow } = require('electron');
+const path = require('path');
+const isDev = require('electron-is-dev');
+const handleFileUrls = require('./handle-file-urls');
 
-let mainWindow = null;
+const emberAppDir = path.resolve(__dirname, '..', 'ember-dist');
+const emberAppURL = pathToFileURL(path.join(emberAppDir, 'index.html')).toString();
```

Comment:
Oh, just a stylistic thing I guess. So many of the methods in the `fs` module are pretty ambiguously named -- `join`, `resolve', `parse`, `normalize` -- and the name of the module is only two characters, so I generally don't destructure it because I find code more readable with the `fs.` prepended everywhere so it's immediately obvious what the method actually is.

Reply:
I agree that path's method names tend to be a bit too vague by themselves. In the past I've done things like `const { join: pathJoin } = requireNode('path')` thinking it might help with optimization / tree shaking / whatever, but after a while I quit as it just seemed to distract me visually and make me think too much.